### PR TITLE
Add user jku and team tuf-root-signing-staging-codeowners

### LINKED ISSUE #347: REQUEST: New membership for jku
### GitHub Username

jku

### Organization you are requesting membership in

sigstore

### Requirements

- [X] I have reviewed the [community membership guidelines](https://github.com/sigstore/community/community-membership.md)
- [X] I have [enabled 2FA on my GitHub account](https://github.com/settings/security)
- [X] I have subscribed to the [sigstore e-mail list](https://groups.google.com/g/sigstore-dev)
- [X] I am actively contributing to 1 or more Sigstore projects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
- [X] I have verified that my sponsors are from different member companies
- [ ] **OPTIONAL:** I have taken the [Inclusive Open Source Community Orientation course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)

### Sponsor 1

@haydentherapper

### Sponsor 2

@kommendorkapten

### List of contributions to the Sigstore project

* Merged PRs to sigstore-python: https://github.com/sigstore/sigstore-python/pulls?q=+is%3Apr+author%3Ajku+is%3Amerged
* I'm already a member of [sigstore-conformance/extremely-dangerous-public-oidc-beacon](https://github.com/sigstore-conformance/extremely-dangerous-public-oidc-beacon)
* Issue for root-signing "refactor" plan: https://github.com/sigstore/root-signing/issues/929

This is part of https://github.com/sigstore/community/issues/345. I plan to provide a PR for this issue (since I'm trying to figure out pulumi anyway for 345).

### PR DESCRIPTION
**Pull request by jku**
* tuf-root-signing-staging-codeowners is for maintainers of planned root-signing-staging repository (see #345)
* New user jku as well as kommendorkapten & haydentherapper are added to the new team

Fixes #347

---

I don't really know what I'm doing with pulumi so feedback is welcome.

### CODE REVIEW

**Review by haydentherapper — COMMENTED** [review]
@cpanato can you confirm if we need to create the team first and then add people? Or if it can be done in one PR?

@jku can you add me as a maintainer also, since I’m currently a maintainer for root signing?

**Comment by cpanato** [comment]
can be togther, worst case we re-run the job :)

**Comment by jku** [comment]
I'll try something... the issues I have based on looking at the existing repository configs
* team id seems to be required (or at least every team definition within a repository has one and https://github.com/sigstore/github-sync README implies I need one)
* Possibly I could use some clever pulumi variable format `id: ${some_team.id}` but I've spent the morning trying to reconcile the yaml used in this repo with the [pulumi github docs](https://www.pulumi.com/registry/packages/github/api-docs/) and ... there seems to be only a vague resemblance between the two? 

I'll come up with something and I guess we'll see what pulumi preview says?

**Comment by jku** [comment]
Armed with some new understanding (that this is not a variant of pulumi yaml, this is a custom format parsed in sisgtore/github-sync), I've tried to come up with a reasonable first configuration for the repository: Some changes will likely be needed later but this could work for now.

currently preview fails with 
`github:index/getUser:getUser: GET https://api.github.com/users/tuf-root-signing-staging-codeowners: 404 Not Found`

File: github-sync/github-data/sigstore/repositories.yaml
Comment (cpanato) [inline github-sync/github-data/sigstore/repositories.yaml:1440]: for this we will need to do in a follow up because we need to know the id for the team

**Comment by jku** [comment]
Cheers. I removed the repository commit, this is now the original branch from two days ago:
* new team tuf-root-signing-staging-codeowners is
* New user jku as well as kommendorkapten & haydentherapper are added to the new team

**Comment by haydentherapper** [comment]
cc @bobcallaway @trevrosen @lukehinds @priyawadhwa @SantiagoTorres for LGTM

**Review by trevrosen — APPROVED** [review]

**Comment by jku** [comment]
still waiting for a merge of this so that I could then add the new repository.


@cpanato when this is merged, will I find the teamID in the workflow output or is there a more involved process to get it?

**Comment by cpanato** [comment]
> still waiting for a merge of this so that I could then add the new repository.
> 
> @cpanato when this is merged, will I find the teamID in the workflow output or is there a more involved process to get it?

you need to get that from the github api, i will do that

**Comment by cpanato** [comment]
but for reference you will need to run

```
gh api /orgs/sigstore/teams/tuf-root-signing-staging-codeowners
```

and get the id :)

**Pull request merged**
